src/main/java/onlinebookstore/exception/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package onlinebookstore.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String s, Exception e) {
```

Comment:
Don’t  use abbreviations 
```suggestion
    public DataProcessingException(String message, Exception exception) {
```